plans/stream.fmf

```diff
@@ -0,0 +1,113 @@
+summary+: |
+    CentOS Stream plans
```

Comment:
Not a blocker but I wonder why do we need to have a separated plan for stream?

Reply:
The idea behind this was to keep it kind of separate either for clarity or readability, Tao this is not a part of a regular suites. We don't necessarily need to run this for each PR. We might attach the plan to a completely different type of trigger in the future.

Reply:
Alright, I just hope it will not create some consistency problems in the future when having separate plans